blockchains/cardano/cardano_worker.js

```diff
@@ -3,16 +3,16 @@
 const got = require('got');
 const uuidv1 = require('uuid/v1');
 const BaseWorker = require('../../lib/worker_base');
-const constants = require('./lib/constants');
 const util = require('./lib/util');
 const { logger } = require('../../lib/logger');
 
 const CARDANO_GRAPHQL_URL = process.env.CARDANO_GRAPHQL_URL || 'http://localhost:3100/graphql';
 const DEFAULT_TIMEOUT_MSEC = parseInt(process.env.DEFAULT_TIMEOUT || '30000');
 
 class CardanoWorker extends BaseWorker {
-  constructor() {
-    super();
+  constructor(constants) {
+    super(constants);
+    this.constants = constants;
```

Comment:
I wonder why we need this if base class already have a field with the same name. Most probably we could access it